diff --git a/config/Config-kernel.in b/config/Config-kernel.in
index a4c46f5..0596124 100644
--- a/config/Config-kernel.in
+++ b/config/Config-kernel.in
@@ -27,7 +27,7 @@ config KERNEL_PRINTK
 config KERNEL_CRASHLOG
 	bool "Crash logging"
 	depends on !(arm || powerpc || sparc || TARGET_uml || i386 || x86_64)
-	default y
+	default n
 
 config KERNEL_SWAP
 	bool "Support for paging of anonymous memory (swap)"
@@ -35,7 +35,7 @@ config KERNEL_SWAP
 
 config KERNEL_DEBUG_FS
 	bool "Compile the kernel with debug filesystem enabled"
-	default y
+	default n
 	help
 	  debugfs is a virtual file system that kernel developers use to put
 	  debugging files into. Enable this option to be able to read and
@@ -62,7 +62,7 @@ config KERNEL_PROFILING
 
 config KERNEL_KALLSYMS
 	bool "Compile the kernel with symbol table information"
-	default y if !SMALL_FLASH
+	default n
 	help
 	  This will give you more information in stack traces from kernel oopses.
 
@@ -107,7 +107,7 @@ config KERNEL_DEBUG_KERNEL
 
 config KERNEL_DEBUG_INFO
 	bool "Compile the kernel with debug information"
-	default y
+	default n
 	select KERNEL_DEBUG_KERNEL
 	help
 	  This will compile your kernel and modules with debug information.
@@ -188,7 +188,7 @@ config KERNEL_BLK_DEV_BSG
 
 config KERNEL_MAGIC_SYSRQ
 	bool "Compile the kernel with SysRq support"
-	default y
+	default n
 
 config KERNEL_DEBUG_PINCTRL
 	bool "Compile the kernel with pinctrl debugging"
@@ -204,7 +204,7 @@ config KERNEL_COREDUMP
 config KERNEL_ELF_CORE
 	bool "Enable process core dump support"
 	select KERNEL_COREDUMP
-	default y
+	default n
 
 config KERNEL_PROVE_LOCKING
 	bool "Enable kernel lock checking"
